### DISCUSSION TITLE
error JSON

### DISCUSSION THREAD

**Discussion by cliff-atmadja** [teamalgoritma/community#113] [Q&A] [answered]
Terdapat error seperti ini, sehingga plot tidak dapat ditampilkan ketika shiny di deploy, kira - kira untuk resolusinya bagaimana ya?
<img width="1440" alt="Screen Shot 2022-05-18 at 23 47 36" src="https://user-images.githubusercontent.com/103408225/169099161-047382e9-31a7-4404-8944-ed2dd0857216.png">

**Comment by NabiilahArdini** [comment, marked as answer]
Halo @cliff-atmadja , dari screenshot Anda sepertinya terdapat 2 pesan: 1 pesan error dan 1 pesan warning.
Pesan terkait JSON sepertinya hanya berupa pesan warning bahwa pada versi selanjutnya fungsi tersebut akan berubah penggunaannya. 
Sepertinya pesan errornya adalah:
```
Input: `..1` must be of size 700 or 1, not 0. 
```
Error ini sering terjadi ketika ketika melakukan filter data, dimana input shiny yang dimasukan akan menghasilkan nol data (tidak ada kondisi yang terpenuhi). Mungkin boleh dicari data Anda yang mencapai jumlah 700 dan apakah terdapat code yang kurang tepat pada bagian filter/pengolahan data tersebut.

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat *mark as answer* jawaban ini.
